model/lifecycle.go

```diff
@@ -373,6 +380,18 @@ func RestartVersion(versionId string, taskIds []string, abortInProgress bool, ca
 		return errors.Wrap(err, "unable to find version")
 	}
 	return errors.Wrap(version.UpdateStatus(evergreen.VersionStarted), "unable to change version status")
+
+}
+
+// RestartVersions restarts selected tasks for a set of versions.
+// If abortInProgress is true for any version, it also sets the abort flag on any in-progress tasks.
+func RestartVersions(taskIds []*VersionToRestart, abortInProgress bool, caller string) error {
```

Comment:
nit. Maybe we should name the first parameter versionsToRestart. Since its an array of objects that contain taskIds